src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -904,6 +932,22 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 			EventHelper.stop(e);
 
 			tab.blur();
+
+			if (isMouseEvent(e) && (e.button !== 0 /* middle/right mouse button */ || (isMacintosh && e.ctrlKey /* macOS context menu */))) {
```

Comment:
Why was this not a problem before?

Reply:
Because there was no code acting on this event.

Maybe this part is actually not required:
`if (e.button === 1) {
					e.preventDefault(); // required to prevent auto-scrolling (https://github.com/microsoft/vscode/issues/16690)
				}`
				
Removing it. The check is still needed